plugin/flags_sources/flags_source.py

```diff
@@ -0,0 +1,60 @@
+"""Holds an abstract class defining a flags source.
+"""
+from os import path
+
+
+class FlagsSource(object):
+    """An abstract class defining a Flags Source.
+    """
+    def __init__(self, include_prefixes):
+        """Initialize default flags storage.
+
+        Args:
+            include_prefixes (str[]): valid include prefixes.
+        """
+        self._include_prefixes = include_prefixes
```

Comment:
As I'm helping with coding sometimes, I'd like to understand the rules of naming that you prefer. It's not always very consistent. Did you mean to use double underscore prefix for private members or single underscore?

And as for public members, you typically define them in the scope of the close, outside of any methods. Is that preferred way for public (and only public) members?

I would personally probably just define all members inside the __init__ function and the prefix would signify whether it's private or public. But I want to follow your style when contributing so that's why I'm trying to make sure. :)